tests/unit/specs/sails-basics.spec.js

```diff
@@ -10,13 +10,13 @@ describe('Sails basics', () => {
     let users = await User.find();
     expect(users.length).toBe(0);
     await User.create({username: 'foo', encryptedPassword: 'xyz123ABCABK7KAL'});
-  
+
     users = await User.find({});
     expect(users.length).toBe(1);
   });
-  
+
   it('should execute sails helpers', async () => {
     const serverIsUp = await sails.helpers.getApiHealth();
     expect(serverIsUp).toBe(true);
   });
-});
+});
```

Comment:
We leave one newline at the end of each file. I should find an eslint rule for this
```suggestion
});

```

Reply:
#983